fixtures/app_with_spm_dependencies/Tuist/Package.swift

```diff
@@ -19,7 +19,8 @@ import PackageDescription
         ],
         projectOptions: [
             "LocalSwiftPackage": .options(disableSynthesizedResourceAccessors: false),
-        ]
+        ],
+        includeLocalPackageTestTargets: true
```

Comment:
Since this is a default, it's not needed. Might be worth adding `false` to some other fixture, though.

Reply:
A lot of integration tests are failing (and were failing) on my machine, let me check the current CI run.